static/scss/custom.scss

```diff
@@ -79,6 +78,20 @@
   color: $hover-color
 }
 
+.btn-purple-rounded {
+  font-weight: bold;
+  background: white;
+  border: 3px solid $hover-color;
+  color: $hover-color;
+  border-radius: 40px;
+  padding: 10px 20px 10px 20px;
+}
+
+.btn-purple-rounded:hover {
+  background: $hover-color;
+  color: white;
+}
+
```

Comment:
btn-purple oraz btn-round to 2 oddzielne klasy utility, ktore powinnas dodawac oddzielnie